### ISSUE TITLE
Phone compatability

### ISSUE THREAD

**Issue by KJ13542** [met4000/clicker.clicker#22]
Please test out your phone compatibility at vision.scibi.com/clickerclicker and comment the model and the problems found.
(Here is a layout to copy).

Mobile Device Brand and Version: (e.g. iPhone 4s, Samsung Galaxy S5, iPad 2 air etc.)
App/Application: (e.g. Google Search, Safari, etc.)
Errors/Bugs/Problems: (e.g. "generic upgrades aren't retaining strikethrough on reload"... etc.)

**Comment by KJ13542** [comment]
To answer your question from Techno, it is mobile compatible except it is hard to click on the cursor since it zooms in every time it is clicked twice quickly.

**Comment by KJ13542** [comment]
Also the layout is messed up for the vision.scibi.com main menu for certain phones.

**Comment by KJ13542** [comment]
Some phones also highlight the clicker in blue preventing you from clicking on it again until the blue highlight is gone so it only allows you to click around 5 times every 3 seconds. (aka not very fast making it boring).

**Comment by KJ13542** [comment]
Also the phone becomes really, really laggy at a certain point. (Obviously, but still worth a mention).

**Comment by met4000** [comment]
Is said point when you have lots and lots of clickers? If so, how many. I may try and find a less resource intensive way... maybe make it less smooth but faster if it thinks it's a phone...

As for double click zooming, I do not believe that can be changed; that is a default action. Also I have made the large cursor un-selectable on PC, but obviously that doesn't work on mobile...

**Comment by met4000** [comment]
Is said point when you have lots and lots of clickers? If so, how many. I may try and find a less resource intensive way... maybe make it less smooth but faster if it thinks it's a phone...

As for double click zooming, I do not believe that can be changed; that is a default action. Also I have made the large cursor un-selectable on PC, but obviously that doesn't work on mobile...

**Comment by met4000** [comment]
Is said point when you have lots and lots of clickers? If so, how many. I may try and find a less resource intensive way... maybe make it less smooth but faster if it thinks it's a phone...

As for double click zooming, I do not believe that can be changed; that is a default action. Also I have made the large cursor un-selectable on PC, but obviously that doesn't work on mobile...

**Comment by met4000** [comment]
Is said point when you have lots and lots of clickers? If so, how many. I may try and find a less resource intensive way... maybe make it less smooth but faster if it thinks it's a phone...

As for double click zooming, I do not believe that can be changed; that is a default action. Also I have made the large cursor un-selectable on PC, but obviously that doesn't work on mobile...

**Comment by met4000** [comment]
Please use this (edited) format:
Mobile Device Brand and Version: (e.g. iPhone 4s, Samsung Galaxy S5, iPad 2 air etc.)
App/Application: (e.g. Google Search, Safari, etc.)
Errors/Bugs/Problems: (e.g. ["generic upgrades aren't retaining strikethrough on reload"](#21)... etc.)

**Comment by met4000** [comment]
Please only phone specific (or if you think it is phone specific) bugs/etc.

**Comment by KJ13542** [comment]
Have you tried making a set zoom point not allowing people to change the zoom?

**Comment by KJ13542** [comment]
Yes 'said point' is having lots of clickers or having a big cps.

**Comment by met4000** [comment]
OK. I'm having a look at how to make it detect mobile devices. I'm currently working on 3.0, so updates might take a bit of time...

**Comment by met4000** [comment]
If you find specific bugs (as opposed to the stated 'general' problems), remember to make a separate issue
